backend/billing/client.go

```diff
@@ -0,0 +1,48 @@
+package billing
+
+import (
+	"io/ioutil"
+	"time"
+
+	"github.com/free5gc/webconsole/backend/logger"
+	"github.com/jlaffaye/ftp"
+)
+
+// The ftp client is for CDR Pull method, that is the billing domain actively query CDR file from CHF
+func FTPLogin() (*ftp.ServerConn, error) {
+	// FTP server is for CDR transfer
+	var c *ftp.ServerConn
+
+	c, err := ftp.Dial("127.0.0.113:2121", ftp.DialWithTimeout(5*time.Second))
```

Comment:
I think the target address should be able to be configured via configuration file.